server/routes/users.js

```diff
@@ -6,4 +6,6 @@ router.get("/user/:id/is_unique", userController.checkUniqueUrl);
 
 router.put("/user/:id", userController.updateUserInfo);
 
+router.post("/signup", userController.signUpUser);
```

Comment:
This is great exactly what I was looking for. I got my basic integration working once this is merged i can perfect it.